lib/src/main/java/uk/gov/di/ipv/cri/passport/library/service/ClientFactoryService.java

```diff
@@ -93,22 +93,22 @@ private HttpClient generateHTTPClientFromExternalApacheHttpClient(
                     KeyManagementException {
 
         String base64TLSCertString =
-                passportConfigurationService.getEncryptedSsmParameter(HTTPCLIENT_TLS_CERT);
+                passportConfigurationService.getEncryptedSsmParameter(DCS_HTTPCLIENT_TLS_CERT);
         Certificate tlsCert = KeyCertHelper.getDecodedX509Certificate(base64TLSCertString);
 
         String base64TLSKeyString =
-                passportConfigurationService.getEncryptedSsmParameter(HTTPCLIENT_TLS_KEY);
+                passportConfigurationService.getEncryptedSsmParameter(DCS_HTTPCLIENT_TLS_KEY);
         PrivateKey tlsKey = KeyCertHelper.getDecodedPrivateRSAKey(base64TLSKeyString);
 
         KeyStore keystoreTLS = createKeyStore(tlsCert, tlsKey);
 
         String base64TLSRootCertString =
-                passportConfigurationService.getEncryptedSsmParameter(HTTPCLIENT_TLS_ROOT_CERT);
+                passportConfigurationService.getEncryptedSsmParameter(DCS_HTTPCLIENT_TLS_ROOT_CERT);
         Certificate tlsRootCert = KeyCertHelper.getDecodedX509Certificate(base64TLSRootCertString);
 
         String base64TLSIntCertString =
                 passportConfigurationService.getEncryptedSsmParameter(
-                        HTTPCLIENT_TLS_INTERMEDIATE_CERT);
+                        DCS_HTTPCLIENT_TLS_INTER_CERT);
         Certificate tlsIntCert = KeyCertHelper.getDecodedX509Certificate(base64TLSIntCertString);
 
         KeyStore trustStore = createTrustStore(new Certificate[] {tlsRootCert, tlsIntCert});
```

Comment:
@smsgds @MMcC-GDS this is quite different to the existing httpclient setup is there a reason for this? Also are we using the sdk clients at the top of this file.
I also see the keystore password is changed. Is that intentional